server/models/schema.sql

```diff
@@ -1958,22 +1962,23 @@ CREATE TABLE `stock_requisition_item` (
 
 DROP TABLE IF EXISTS `stock_movement`;
 CREATE TABLE `stock_movement` (
-  `uuid`            BINARY(16) NOT NULL,
-  `document_uuid`   BINARY(16) NOT NULL,
-  `depot_uuid`      BINARY(16) NOT NULL,
-  `lot_uuid`        BINARY(16) NOT NULL,
-  `entity_uuid`     BINARY(16) NULL, -- holds the patient_uuid, destination depot_uuid, service_uuid, or purchase_order uuid
-  `description`     TEXT NULL,
-  `flux_id`         INT(11) NOT NULL,
-  `date`            DATETIME NOT NULL,
-  `quantity`        INT(11) NOT NULL DEFAULT 0,
-  `unit_cost`       DECIMAL(19, 4) UNSIGNED NOT NULL,
-  `is_exit`         TINYINT(1) NOT NULL,
-  `user_id`         SMALLINT(5) UNSIGNED NOT NULL,
-  `reference`       INT(11) UNSIGNED NOT NULL,
-  `invoice_uuid`    BINARY(16) NULL,
-  `period_id`       MEDIUMINT(8) UNSIGNED NOT NULL,
-  `created_at`      TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `uuid`                    BINARY(16) NOT NULL,
+  `document_uuid`           BINARY(16) NOT NULL,
+  `depot_uuid`              BINARY(16) NOT NULL,
+  `lot_uuid`                BINARY(16) NOT NULL,
+  `entity_uuid`             BINARY(16) NULL, -- holds the patient_uuid, destination depot_uuid, service_uuid, or purchase_order uuid
+  `description`             TEXT NULL,
+  `flux_id`                 INT(11) NOT NULL,
+  `date`                    DATETIME NOT NULL,
+  `quantity`                INT(11) NOT NULL DEFAULT 0,
+  `unit_cost`               DECIMAL(19, 4) UNSIGNED NOT NULL,
+  `is_exit`                 TINYINT(1) NOT NULL,
+  `user_id`                 SMALLINT(5) UNSIGNED NOT NULL,
+  `reference`               INT(11) UNSIGNED NOT NULL,
+  `invoice_uuid`            BINARY(16) NULL,
+  `stock_requisition_uuid`  BINARY(16) NULL,
```

Comment:
In the long run, we could probably merge the columns for `invoice_uuid` and _`stock_requisition_uuid`.